modules/core/src/matrix_transform.cpp

```diff
@@ -282,6 +283,69 @@ void transpose( InputArray _src, OutputArray _dst )
 }
 
 
+
+template <typename T>
+struct transposeImpl
+{
+    void operator()(const cv::Mat& inp, cv::Mat& out, const std::vector<int>& order) const
+    {
+        int continuous_idx = 0;
```

Comment:
continuous vs contiguous

Reply:
Isn't it the same thing? Since we have `cv::Mat::isContinuous()`, I used `continuous_idx`.